config/eslint.js

```diff
@@ -108,4 +108,13 @@ module.exports = {
     sourceType: 'module',
     project: './tsconfig.json',
   },
+  overrides: [
+    {
+      files: ['test/**/*.ts', 'tests/**/*.ts'],
+      rules: {
+        'implicit-dependencies/no-implicit': 'off',
```

Comment:
Are there any risks you are aware of turning these off in tests?  I'm assuming this is here since the linter throws errors about tape not being present (since it's moving to the root)?